test/integration/targets/postgresql/tasks/pg_hba.yml

```diff
@@ -0,0 +1,133 @@
+- name: Make sure file does not exist
+  file:
+    dest: /tmp/pg_hba.conf
+    state: absent
+- name: check_mode run
+  pg_hba:
+    dest: /tmp/pg_hba.conf
+    contype: host
+    source: '0000:ffff::'
+    netmask: 'ffff:fff0::'
+    method: md5
+    backup: true
+    order: sud
+    state: "{{item}}"
+  check_mode: yes
+  with_items:
+  - present
+  - absent
+- name: check_mode check
+  stat:
+    path: /tmp/pg_hba.conf
+  register: pg_hba_checkmode_check
+- name: Remove several ip addresses for idempotency check
```

Comment:
Because `/tmp/pg_hbd.conf` doesn't exist and `state` is `absent`, should not the module return `changed == False` (meaning: `pg_hba_idempotency_check1 is not changed`) ?

Reply:
This was already the case, but because 'changed: true'. This now also is the case with changed: False.